docs/kusama-parameters.md

```diff
@@ -5,13 +5,13 @@ sidebar_label: Parameters
 ---
 
 Many of these parameter values can be updated via on-chain governance. If you require absolute
-certainty as to their values, it is recommended you directly check the constants by looking at the
+certainty of these parameter values, it is recommended you directly check the constants by looking at the
 [chain state](https://polkadot.js.org/apps/#/chainstate/constants) and/or
 [storage](https://polkadot.js.org/apps/#/chainstate).
 
 ### Periods of common actions and attributes
 
-_NOTE: Kusama generally runs 4x as fast as Polkadot, except Polkadot also has 6 second slots. See
+_NOTE: Kusama generally runs 4x as fast as Polkadot, where Polkadot has a slot time duration of 6 seconds. See
```

Comment:
I don't understand this change or why it's necessary. Generally things happen 4x as fast on Kusama as compared to Polkadot, EXCEPT in slot duration time itself.

Reply:
I understand, thanks! I interpreted differently.